posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Introducing the Open Liberty Starter!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/steven1046
+author_github: https://github.com/steven1046
+seo-title: Yet another way to get started with Open Liberty - OpenLiberty.io
+seo-description: Yet another way to get started with Open Liberty
+blog_description: Yet another way to get started with Open Liberty
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Introducing the Open Liberty Starter!
+Steven Zvonek <https://github.com/steven1046>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+== Open Liberty Starter
+
+There is now another easy way to get started using Open Liberty. Visit the link:{url-prefix}/downloads/[Getting Started] page and select a few options to generate a starter application using Open Liberty. You can specify your application and project name, choose a build tool from either Maven or Gradle, and pick which version of Java SE, Jakarta EE, and MicroProfile your application will use. Then, just click `Generate Project` and you are ready for lift-off! 
+
+Once you download the starter project, unpackage the .zip file on your machine.
+
+If you selected `Maven` as your build tool, then navigate to the installation directory and run `mvn liberty:run`. This will install all required dependencies and start the default server. 
+
+If you selected `Gradle` as your build tool, then navigate to the installation directory and run `./gradlew libertyStart` to start the default server. 
+
+For more information on developing your application in dev mode using either Maven or Gradle, see the link:{url-prefix}/docs/latest/development-mode.html[dev mode documentation].
+
+For further help on getting started actually developing your application, see some of our link:{url-prefix}/guides/?search=microprofile&key=tag[Microprofile guides] and link:{url-prefix}/guides/?search=jakarta%20ee&key=tag[Jakarta EE guides].
+
+
+[link={url-prefix}/downloads]
```

Comment:
I think we should move this image up to right after your first paragraph (aka, after the 'then just click generate' part.